### PR TITLE
doc(bpmnvisu_monitoring): Update monitoring page with new BPMN visu

### PR DESCRIPTION
**Pull request by jeromecambon**
Note: PR https://github.com/bonitasoft/bonita-doc/pull/2083 should be merged first, since this PR reference a new page from this one. Then xrefs should be uncommented.

Covers [RUNTIME-1212](https://bonitasoft.atlassian.net/browse/RUNTIME-1212)

### CODE REVIEW

File: modules/runtime/pages/cases.adoc [hunk 1]
```diff
@@ -37,11 +37,16 @@ If you want to go back to the default view settings, clear your browser cache an
 
 From this table, you can delete a case or several cases. Select the cases you want to delete, and click on _Delete_. Confirm in the popup that you want to delete the selected cases.
 
-if you are using the Enterprise, Performance or Efficiency edition, you can also open the diagram of a case (case visualization), by clicking on the _Picture_ icon for the case.
-On the diagram, colored flags on the steps show the number of step instances in the possible states. Click on _Show key_ to see an explanation of the display.
+if you are using the Enterprise, Performance or Efficiency edition, you can also open the diagram of a case (case visualization), by clicking on the _Picture_ icon for the case. +
+On the diagram, colored flags on the steps show the number of step instances in the possible states. +
```
Comment (abirembaut) [inline modules/runtime/pages/cases.adoc]:
```suggestion
On the diagram, colored flags on the steps show the number of step instances in each state. +
```

File: modules/runtime/pages/cases.adoc [hunk 2]
```diff
@@ -37,11 +37,16 @@ If you want to go back to the default view settings, clear your browser cache an
...
+Put your mouse hover any node to see an explanation of the display. +
```
Comment (abirembaut) [inline modules/runtime/pages/cases.adoc]:
```suggestion
Put your mouse hover any step to see an explanation of the display. +
```
for consistency with the previous sentence

File: modules/runtime/pages/cases.adoc [hunk 3]
```diff
@@ -37,11 +37,16 @@ If you want to go back to the default view settings, clear your browser cache an
...
+Zoom in the diagram with either the mouse wheel or the zoom buttons. Once zoomed in, you can drag the diagram. +
+Resize your browser window and click on the *_Reset_* button to fit the diagram to your window.
+
 Remember that a failure at a step does not necessarily mean that the case has failed entirely, but it might indicate that some action is needed, perhaps to xref:ROOT:admin-application-task-list.adoc#_replay_a_failed_task[replay a connector].
 
-The process diagram is constructed from the deployed process definition. This means that some details visible in Bonita Studio, such as annotations, are not available.
+You can customize the diagram page, or embed the BPMN diagram in your own page: see <next line to uncomment before merge>
+//xref:ROOT:customize-display-process-monitoring.adoc[How to customize the display of process execution monitoring].
```
Comment (abirembaut) [inline modules/runtime/pages/cases.adoc]:
```suggestion
You can customize the diagram page, or embed the BPMN diagram in your own page: see xref:pages-and-forms:customize-display-process-monitoring.adoc[How to customize the display of process execution monitoring].
```

File: modules/runtime/pages/monitoring.adoc [hunk 4]
```diff
@@ -28,15 +28,18 @@ You can filter the table to show only a specified process, or process and versio
 Process Managers can see only the processes they manage.
 
 From a row in the table, you can click the _List_ icon to see a list of cases of the process.
-You can also open a diagram of a process, by clicking on the _Picture_ icon for the process. On the diagram, you can see the number and state of cases in progress at each step.
-Click on the _Show key_ button to see an explanation of the display.
 
-The process diagram is constructed from the deployed process definition. This means that some details visible in Bonita Studio, such as descriptions are not available.
+You can also open a diagram of a process, by clicking on the _Picture_ icon for the process. On the diagram, you can see the number and state of cases in progress at each step. +
```
Comment (abirembaut) [inline modules/runtime/pages/monitoring.adoc]:
```suggestion
You can also open the diagram of a process, by clicking on the _Picture_ icon for the process. On the diagram, you can see the number and state of cases in progress at each step. +
```

File: modules/runtime/pages/monitoring.adoc [hunk 5]
```diff
@@ -28,15 +28,18 @@ You can filter the table to show only a specified process, or process and versio
...
+Put your mouse hover any node to see an explanation of the display. +
```
Comment (abirembaut) [inline modules/runtime/pages/monitoring.adoc]:
```suggestion
Put your mouse hover any step to see an explanation of the display. +
```

File: modules/runtime/pages/monitoring.adoc [hunk 6]
```diff
@@ -28,15 +28,18 @@ You can filter the table to show only a specified process, or process and versio
...
+Zoom in the diagram with either the mouse wheel or the zoom buttons. Once zoomed in, you can drag the diagram. +
+Resize your browser window and click on the *_Reset_* button to fit the diagram to your window.
+
+You can customize the diagram page, or embed the BPMN diagram in your own page: see <next line to uncomment before merge>
+//xref:ROOT:customize-display-process-monitoring.adoc[How to customize the display of process execution monitoring].
```
Comment (abirembaut) [inline modules/runtime/pages/monitoring.adoc]:
```suggestion
You can customize the diagram page, or embed the BPMN diagram in your own page: see xref:pages-and-forms:customize-display-process-monitoring.adoc[How to customize the display of process execution monitoring].
```

**Review by abirembaut — CHANGES_REQUESTED** [review]
just a few suggestions

**Review by vbotnari12 — CHANGES_REQUESTED** [review]
Please mention here about the zoom limitations with the magic mouse.

**Pull request merged**